src/main/java/com/checkmarx/jenkins/CxScanBuilder.java

```diff
@@ -2481,8 +2484,8 @@ public FormValidation doCheckForceScan(@QueryParameter boolean value, @QueryPara
          * @param value
          * @return
          */
-        public FormValidation doCheckIncremental(@QueryParameter boolean value, @QueryParameter boolean forceScan) {
-            Jenkins.getInstance().checkPermission(Item.CONFIGURE);
+        public FormValidation doCheckIncremental(@QueryParameter boolean value, @QueryParameter boolean forceScan,@AncestorInPath Item item) {
+                item.checkPermission(Item.CONFIGURE);
```

Comment:
```suggestion
                if (item == null) {
                        return FormValidation.ok();
                }
                item.checkPermission(Item.CONFIGURE);
```
Please check if the context is not null